src/hooks/chatSidebar.ts

```diff
@@ -83,3 +89,24 @@ export const useChatSidebarInfo = () => {
     venueChatTabTitle: `${chatTitle} Chat`,
   };
 };
+
+export const useChatSidebarPinned = () => {
+  const dispatch = useDispatch();
+  const { width } = useWindowSize();
+  const venue = useSelector(currentVenueSelectorData);
+  const isPinned = useMemo(
+    () =>
+      width > LARGE_SCREEN_WIDTH && venue?.template !== VenueTemplate.audience,
```

Comment:
It would probably be good to extract this into a `const`, maybe at the top of this file, maybe in `src/settings.ts` (obviously come up with a better name than `FOO_GOOD_CONST_NAME`):

```suggestion
      width > LARGE_SCREEN_WIDTH && !FOO_GOOD_CONST_NAME.includes(venue?.template),
```

Reply:
@yurii-lubynets that const will be an array with all(currently it's just `audience`, but will be more in the future) venues that need `chatsidebar` to be always pinned

Reply:
fixed, thank you a lot